libr/anal/fcn.c

```diff
@@ -3,13 +3,15 @@
 #include <r_anal.h>
 #include <r_util.h>
 #include <r_list.h>
+#include <r_flags.h>
```

Comment:
Ranal doesnt depends on flags in the makefile.. And i would prefer not to add it. Use any of the binded helpers